### DISCUSSION TITLE
Ability to export data

### DISCUSSION THREAD

**Discussion by seailz** [diced/zipline#561] [Ideas]
I want to move zipline to another VPS, but keep all of my files to ensure pre-existing links all still function correctly. Not sure if there's already a way to do this or not

**Comment by diced** [comment]
there is a export data feature already, go to settings and scroll down it should be there.

but if you wanted to move your data to another server, you should just use `rsync` to transfer the files directory, then you could follow this guide on how to migrate postgres to a different version (just ignore the stuff about the version and just focus on the postgres backup / restore) https://zipline.diced.sh/docs/guides/migrate-pg-14-15. doing the postgres backup and restore will just make it so you have all your old data on the new server, passwords, users, etc will all work.

if you don't want to do that you can use the import-dir script https://zipline.diced.sh/docs/guides/importing after transferring the files using `rsync` then recreate users and such.
